test/hybig/HyBIG_Regression.ipynb

```diff
@@ -0,0 +1,428 @@
+    "        expected_output_metadata = {\n",
+    "            'driver': 'PNG', **common_measures_metadata\n",
+    "        }\n",
+    "\n",
+    "        assert_dataset_produced_correct_results(\n",
+    "            test_files[0], expected_output_metadata, reference_files[0], 'PNG'\n",
+    "        )\n",
+    "\n",
+    "    print_success('Conversion of MEaSUREs Geotiff to PNG Success')\n",
+    "else:\n",
+    "    print('Skipping test: HyBIG regression tests not configured for this environment.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab9d4370",
+   "metadata": {},
+   "source": [
+    "### Test a request for JPEG output from 3-Band RGB input data\n",
+    "\n",
+    "Forms a request for MEaSUREs VCF5KYR data (3-band RGB GeoTiff) as 'image/jpg', ensures all files are created that the image has the correct metadata and that that the data in the JPEG file matches the reference data in the test."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56bb7ba2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if environment_information is not None:\n",
+    "    measures_request = Request(collection=environment_information['measures_collection'],\n",
+    "                               granule_id=environment_information['measures_granules'][0],\n",
+    "                               format='image/jpeg')                       \n",
+    "\n",
+    "    measures_job_id = harmony_client.submit(measures_request)\n",
+    "    harmony_client.wait_for_processing(measures_job_id, show_progress=True)\n",
+    "\n",
+    "    reference_files = build_file_list(measures_base, reference_dir, 'JPG')\n",
+    "    \n",
+    "    with TemporaryDirectory() as temp_dir:\n",
+    "        downloaded_grid_outputs = [\n",
+    "            file_future.result()\n",
+    "            for file_future\n",
+    "            in harmony_client.download_all(measures_job_id, overwrite=True, directory=temp_dir)\n",
+    "        ]\n",
+    "        \n",
+    "        test_files = build_file_list(measures_base, Path(temp_dir), 'JPG')\n",
+    "        for file_name in test_files:\n",
+    "            assert file_name.exists() == True, f'File does not exist {file_name.resolve()}'\n",
+    "        print_success('all test files generated')\n",
+    "\n",
+    "        expected_output_metadata = {\n",
+    "            'driver': 'JPEG', **common_measures_metadata\n",
+    "        }\n",
+    "\n",
+    "        assert_dataset_produced_correct_results(\n",
+    "            test_files[0], expected_output_metadata, reference_files[0], 'JPEG'\n",
+    "        )\n",
+    "\n",
+    "    print_success('Conversion of MEaSUREs Geotiff to JPEG Success')\n",
+    "else:\n",
+    "    print('Skipping test: HyBIG regression tests not configured for this environment.')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "132e50db",
+   "metadata": {},
+   "outputs": [],
+   "source": []
```

Comment:
One last _super_ nit: This empty cell at the end of the notebook could be removed.

Reply:
[659c4d4](https://github.com/nasa/harmony-regression-tests/pull/63/commits/659c4d484ebae4d82177f86e285e874031231a25)